src/components/Book.js

```diff
@@ -14,23 +14,27 @@ const Book = (props) => {
 
```

Comment:
- [ ] You are almost there, @tajemouti :clap: I think you just have to check your `BookPage` again in order to make it conform with the project requirement, and how you can fix the following things:
  - [ ] The name of the component my be better of if it maintains the suggested `<BookList />`
  - [ ] The requirement asked that you fetchBook using a `useEffect` hook to manage the post-event behaviour as shown below :point_down: 
![image](https://github.com/tajemouti/bookstore/assets/99715640/ac8e4a85-e86c-4b0c-97e2-e8f38b35d650)

Reply:
Hi @AbayomiOlaoye,
Thank you for taking the time to review my code. I really appreciate it. But I can't see those specific requirements you pointed out in your feedback on the project requirements.
![Capture d'écran 2023-09-01 031122](https://github.com/tajemouti/bookstore/assets/130588108/3313378b-c8f5-4050-9014-77da3522567c)
Here's the [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connect_to_api_v2_1.md) to this project page.

Reply:
Oh, yes! Sorry about that but I those files are not among the recent changes you made. The best way to acces that will be to look at your file directory - `BookPage` is in your components folder

Reply:
It's alright @AbayomiOlaoye. However, I think that my code aligns with the project requirements, and the changes you required are not listed in the project requirements. The Book component contains the book list element markups.
Cheers